lib/blocks.tsx

```diff
@@ -0,0 +1,95 @@
+import { createElement, ReactNode } from 'react';
```

Comment:
now that we fixed the Jest issue, pretty sure this file doesn't need to be `.tsx` because it doesn't contain a JSX expression (`<Component />`)

Reply:
@chriszarate I think it was because of these lines, but I can remove it. https://github.com/Automattic/vip-go-nextjs-skeleton/pull/68/files/229981096c4f7d4e23a8fdb7a9071d0509d77f3a#diff-6a4a0c2d9670bcb1dd7fa952353e2da18a08f4733173f55d72658d3e7e800851R62

Reply:
Ah you're right, that is JSX.